### DISCUSSION TITLE
Plex support or only Jellyfin?

### DISCUSSION THREAD

**Discussion by khorinisgefasel** [arianneorpilla/jidoujisho#396] [Q&A]
Hi all!
Is it also possible to use Plex or only Jellyfin?

Big thanks in advance!

Great app!!!
Thank you a lot!

**Comment by brishtibheja** [comment]
Did plex work when trying or you read plex works here?

**Comment by khorinisgefasel** [comment]
I asked if it works. I haven't tried it myself. 
In the tutorial it is not mentioned, but it used to be mentioned in an early version.
So does anyone know if it gets supported?

**Comment by brishtibheja** [comment]
The legacy `README.md` does mention it. Here a code search showed [this intent filter](https://github.com/arianneorpilla/jidoujisho/blob/2de1514f25f7d78619d8f8d33761e2816b13637a/yuuna/android/app/src/main/AndroidManifest.xml#L71). I assume it does although I'd rather try it first instead of concluding from searches. The natural thing to assume when documentation is removed is that it got irrelevant.
